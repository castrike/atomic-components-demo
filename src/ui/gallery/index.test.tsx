import {
  render,
} from '@testing-library/react';
import { Gallery } from '.';

describe('Testing Sneaker Box', () => {
  test('Renders', () => {
    const { container } = render(<Gallery>
      <p>test</p>
      <p>test</p>
      <p>test</p>
    </Gallery>)
    expect(container).toMatchSnapshot();
  })
})