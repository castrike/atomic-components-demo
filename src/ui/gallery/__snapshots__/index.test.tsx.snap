// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Sneaker Box Renders 1`] = `
<div>
  <ul
    class="grid grid-cols-3 max-w-xl mx-auto gap-4"
  >
    <li>
      <p>
        test
      </p>
    </li>
    <li>
      <p>
        test
      </p>
    </li>
    <li>
      <p>
        test
      </p>
    </li>
  </ul>
</div>
`;
